packages/lb-components/src/components/longText/index.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file Text exceeds display view more
+ * @author lixinghua <lixinghua@sensetime.com>;
+ * @date 2023-04-10
+ */
+
+import React, { useRef, ReactElement, CSSProperties } from 'react';
+import { Popover, Tooltip } from 'antd';
+import { useSize } from 'ahooks';
+import { TooltipPlacement } from 'antd/es/tooltip';
+import { prefix } from '@/constant';
+import { SearchOutlined } from '@ant-design/icons';
+
+interface IProps {
+  text: string;
+  placement?: TooltipPlacement; // Popover位置
+  icon?: ReactElement | null; // 自定义icon
+  openByText?: boolean; // hover文字展开
+  style?: CSSProperties;
+  isToolTips?: boolean;
+  wordCount?: number;
+}
+
+const longTextCls = `${prefix}-longText`;
+const LongText = (props: IProps) => {
+  const {
+    text,
+    placement = 'bottom',
+    icon = <SearchOutlined />,
+    openByText,
+    style,
+    isToolTips,
+    wordCount,
+  } = props;
+
+  const el = useRef<null | HTMLDivElement>(null);
+  useSize(el);
```

Comment:
What is the purpose of using `useSize` ?

Reply:
Used to determine if the label content is overflowing